content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,809 @@
+activities or emails configured for this stage, unlike for the :guilabel:`First Interview` stage.
+The recruiter can now :ref:`schedule a second interview <recruitment/schedule>` with the applicant,
+following the same process as for the first interview.
+
+Contract proposal
+-----------------
+
+When the applicant has passed the interview stages and an offer is ready to be sent, they can be
+moved to the :guilabel:`Contract Proposal` stage. Drag and drop the applicant's card to the
+:guilabel:`Contract Proposal` stage, or click on the :guilabel:`Contract Proposal` stage button at
+the top right of the individual applicant's card.
+
+The next step is to send an offer to the applicant. On the applicant's card, click the
+:guilabel:`Generate Offer` button. A :guilabel:`Generate a Simulation Link` pop-up appears. All
+fields are pre-populated with information from the job position, except for the :guilabel:`Force
+New Cars List`, :guilabel:`Default Vehicle`, and the :guilabel:`Canteen Cost` (this is set to
+`0.00`).
+
+.. note::
+   Not all fields may appear on the :guilabel:`Generate a Simulation Link` pop-up. Depending on the
+   localization setting for the company and the applications installed, some fields may not appear.
+   For example if the *Fleet* application is not installed, the fields related to vehicles will not
+   appear on the pop-up.
+
+Universal fields
+~~~~~~~~~~~~~~~~
+
+The following fields appear for all offers sent to applicants regardless of localization settings.
+
+- :guilabel:`Offer Template`: The template currently being used to populate the :guilabel:`Generate
+  a Simulation Link` pop-up. To modify the template, click the :guilabel:`External link` icon to the
+  right of the drop-down arrow. Make any changes, then click :guilabel:`Save`.
+- :guilabel:`Job Title`: The name of the position being offered to the applicant.
+- :guilabel:`Contract Start Date`: The date the contract takes effect. The default date is the
+  current date. To modify the date, click the drop-down to reveal a calendar. Use the :guilabel:`<
+  (left)` and :guilabel:`> (right)` arrows on either side of the month to navigate to the desired
+  month, then click on the :guilabel:`day` to select the date.
+- :guilabel:`Yearly Cost`: The annual salary being offered.
+- :guilabel:`Offer Link`: This link takes the applicant to a webpage where they can enter their
+  personal information, which will be imported to their employee record when created. If applicable,
+  the applicant can modify their salary package (this option is not available for all
+  localizations). Lastly, this is where the applicant accepts the offer by clicking
+  :guilabel:`Review Contract & Sign` to accept the contract and sign it using the *Sign*
+  application.
+
+.. note::
+   The *Sign* application must be installed in order for an applicant to sign any document,
+   including a contract. Ensure the *Sign* application is installed to use this feature.
```

Comment:
This sounds like the applicant needs the sign app

Reply:
I changed the wording so it doesn't say "applicant" and implies only the user needs the sign app (which is correct).